assets/js/components/shapeshift/shift-confirm.component.js

```diff
@@ -30,6 +30,17 @@ function ShiftConfirmController (AngularHelper, $scope, Exchange, Wallet, Ethere
   $scope.bitcoinCash = currency.bchCurrencies.filter(c => c.code === 'BCH')[0];
   $scope.getTimeToExpiration = () => $scope.quote.expires - now;
 
+  $scope.totalLeavingWallet = () => {
```

Comment:
Could we move the bitCurrencyMap to the checkout controller and use that here as well?

$scope.bitCurrencyMap = {
    'eth': { currency: $scope.ether, convert: currency.convertToEther, icon: 'icon-ethereum' },
    'btc': { currency: $scope.bitcoin, convert: currency.convertToSatoshi, icon: 'icon-bitcoin' },
    'bch': { currency: $scope.bitcoinCash, convert: currency.convertToBitcoinCash, icon: 'icon-bitcoin-cash' }
  };